app/controllers/users/registrations_controller.rb

```diff
@@ -38,7 +38,6 @@ def import_users
   # POST /resource
    def create
         if params[:admin]
-            puts 'hi'
         end
```

Comment:
Let's go ahead and remove this whole block, since it's empty now and that's not how we should be setting admin stuff anyhow!